doc/nrf/known_issues.rst

```diff
@@ -200,6 +200,13 @@ NCSDK-20047: SLM logging over RTT is not available
 Other issues
 ============
 
+.. rst-class:: v2-3-0
+
+CIA-892: Assert when printing strings with the ``%s`` qualifier
+  When logging a string with the ``%s`` qualifier, the maximum length of any inserted string is 1023 bytes.
+  If the string is longer, an assert or a crash will occur.
```

Comment:
It's not certain that a crash will happen. The stack is corrupted, so the subsequent behavior is undefined

Reply:
Changed "will" to "can".